modules/cql/src/blaze/elm/ratio.clj

```diff
@@ -0,0 +1,52 @@
+(ns blaze.elm.ratio
+  "Implementation of the ratio type."
+  (:require
+    [blaze.elm.protocols :as p]
+    [blaze.elm.quantity :as quantity])
+  (:import (javax.measure UnconvertibleException)))
```

Comment:
We use the same vector-based syntax for imports as for requires. also with newline after `:import`.